### ISSUE TITLE
Documentation: Working with Prow from PR author perspective

### ISSUE THREAD

**Issue by michal-hudy** [kyma-project/test-infra#72]
**Description**

User guide for Prow from PR creator/reviewer perspective. 

**Reasons**

Well described process.

**Attachments**

**Comment by michal-hudy** [comment]
@kazydek I think we can start working on it 😄

**Comment by aszecowka** [comment]
In documentation, IMO the following questions should be answered:
- what is triggered when PR is created?
- how can I see the output from a test?
- how can I retrigger job?
here we can describe that all available commands are documented here:
https://status.build.kyma-project.io/command-help?repo=kyma-project%2Fkyma
- what is triggered when PR is merged to the master branch?

**Comment by kazydek** [comment]
This task relates to the issue https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/issues/kyma-project/test-infra/337

**Comment by kazydek** [comment]
Ok, I created https://github.com/kyma-project/community/pull/175 with the requested changes. I also included AC from https://github.com/kyma-project/test-infra/issues/337 in it. 
@aszecowka , I haven't included the link to https://status.build.kyma-project.io/command-help?repo=kyma-project%2Fkyma however as it includes incorrect descriptions. However, all test comments are described in the CONTRIBUTING.md. I also did not describe which jobs are triggered specifically when a PR is created or what a merge triggers as that will be covered in https://github.com/kyma-project/test-infra/issues/169 (already listed in the AC).
